SU-HKKU/cardinal_service_profile
# CSD-440  Created duration range block view for opportunity content type

# READY FOR REVIEW

# Summary
- Created duration range block view for opportunity content type

# Needed By (Date)
- 09/11

# Urgency
- mediuma

# Steps to Test

1. Pull this PR
2. Clear cache
3. View any opportunity. The When should display the relative duration time.
_*Note there's some funky margin-bottom spacing going on which will be resolved in another PR_

# Associated Issues and/or People
- [CSD-440](https://stanfordits.atlassian.net/browse/CSD-440)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
